src/webextension/list/manage/components/account-summary.js

```diff
@@ -20,7 +20,7 @@ export function AccountSummaryButton({displayName, avatar, onClick}) {
   return (
     <button className={styles.button} onClick={onClick}>
       <span className={styles.displayName}>{displayName}</span>
-      <span className={styles.avatar}><img src={avatar}/></span>
+      <span className={styles.avatar}><img alt="" src={avatar}/></span>
```

Comment:
The purpose of alternative text is to describe the purpose or content of images to screen reader users. Could you add "user avatar" or something similar here to explain what the image is?